plugin/db/pg/pg.go

```diff
@@ -357,41 +357,44 @@ func (driver *Driver) getUserList(ctx context.Context) ([]*db.User, error) {
 }
 
 // Execute executes a SQL statement.
-func (driver *Driver) Execute(ctx context.Context, statement string, useTransaction bool) error {
-	// We don't use transaction for creating databases in Postgres.
-	// https://github.com/bytebase/bytebase/issues/202
-	if !useTransaction {
-		f := func(stmt string) error {
-			// For the case of `\connect "dbname";`, we need to use GetDbConnection() instead of executing the statement.
-			if strings.HasPrefix(stmt, "\\connect ") {
-				parts := strings.Split(stmt, `"`)
-				if len(parts) != 3 {
-					return fmt.Errorf("invalid statement %q", stmt)
-				}
-				_, err := driver.GetDbConnection(ctx, parts[1])
-				return err
-			}
+func (driver *Driver) Execute(ctx context.Context, statement string) error {
+	var remainingStmts []string
+	f := func(stmt string) error {
+		if strings.HasPrefix(stmt, "CREATE DATABASE ") {
```

Comment:
Maybe a minor improvement is to strip the leading empty spaces

Reply:
This is actually intended to prevent other cases with the same prefix such as "CREATE DATABASE_XXX".

Reply:
I mean the leading empty space instead of the ending empty spaces.

Reply:
Ah! Since the "CREATE DATABASE" statement is generated by us, it doesn't have the leading spaces. Also driver.db.ExecContext() should be able to handle the spaces.

Reply:
>> Since the "CREATE DATABASE" statement is generated by us

I guess only for now, but this module will be used by the library which is out of our control, so an extra guide doesn't hurt.

>> Also driver.db.ExecContext() should be able to handle the space

we will delve into the tx route, which will fail

Reply:
Done.